tests/ethpm/test_dependencies.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+
+from ethpm import Package
+from ethpm.dependencies import Dependencies
+from ethpm.exceptions import UriNotSupportedError
+from ethpm.validation import validate_build_dependencies
+
+
+@pytest.fixture
+def dependencies(monkeypatch, standard_token_manifest, piper_coin_manifest):
+    monkeypatch.setenv(
+        "ETHPM_IPFS_BACKEND_CLASS", "ethpm.backends.ipfs.DummyIPFSBackend"
```

Comment:
You might make a fixture that does this so that you can easily re-use it.

```python
@pytest.fixture
def dummy_ipfs_backend(monkeypatch):
    ... # do the monkeypatch here

@pytest.fixture
def dependencies(dummy_ipfs_backend, ...):
    # anything that uses this dependency will also automatically monkeypatch the ipfs backend)
    ...
```